rasa/core/channels/twilio_voice.py

```diff
@@ -148,23 +148,26 @@ def _validate_configuration(self) -> None:
         try:
             int(self.speech_timeout)
         except ValueError:
-            if self.speech_timeout.lower() != "auto":
+            if self.speech_timeout and self.speech_timeout.lower() != "auto":
```

Comment:
> Do you know if the class can actually accept None for this attribute?
Thought it's best to check in with @mvielkind on this one, do you know if it's alright for these parameters to be of type `Text` only and accept defaults to avoid `union-attr` mypy errors in this method `_validate_configuration`?

Reply:
@mvielkind Sorry to chase, could you please have a look at the change in this file and let me know if it looks good to you?